### DISCUSSION TITLE
HELP:  Importing new app error

### DISCUSSION THREAD

**Discussion by Mattys556** [Budibase/budibase#15269] [Help] [answered]
Thanks for any help, I am completely lost here. 

I hired a guy on Upwork to build me an app and he did a great job, works perfect, but I needed to own the app, so he send me a Tar.gz file... No matter what I do the import app gives me a error. 

I have tried extracting the fike, no idea what I am doing, but also keep getting errors on importing.

**Comment by mikesealey** [comment, marked as answer]
No need to extract the file, simply import the entire .tar.gz

From your account portal, go to apps (Top left corner)
![image](https://github.com/user-attachments/assets/fe2b10b6-3ff9-4a77-8f0d-59ceb152f8cf)

If you already have apps in your account portal, you'll be greeted with this:
![image](https://github.com/user-attachments/assets/5fe51f9e-df4c-40ee-8357-5a8be8c2d987)

If not, you'll find yourself here: 
![image](https://github.com/user-attachments/assets/4303cdfa-a8b9-46bc-8045-bd80d0272f98)
You can step through creating "My First App, and then use the back (left) arrow to get back to the account portal
![image](https://github.com/user-attachments/assets/c6dcfd36-9a83-4667-b485-f0aff0c3ecc2)

Then click "Create New App", and "Import App"
![image](https://github.com/user-attachments/assets/8ea7d474-f14b-44b2-be05-7c57552659bb)

From there you can import the .tar.gz file
![image](https://github.com/user-attachments/assets/925a04ad-0bf0-46b0-9153-53fc32f1ca6d)

And that's it! You should be up and running with your new app. If you need any further instruction, head to the [documentation](https://docs.budibase.com/docs/export-and-import-apps#import-an-application) or if you're looking for our community on [Discord head here](https://discord.com/invite/ZepTmGbtfF).

**Reply by Mattys556** [reply]
Thats exactly what I am doing and I get an error code everytime.

**Reply by Mattys556** [reply]
I keep getting " Failed to update app - [object Object]"
